Source/MarkdownGenerator/src/main/java/net/steppschuh/markdowngenerator/list/OrderedList.java

```diff
@@ -0,0 +1,29 @@
+package net.steppschuh.markdowngenerator.list;
+
+import java.util.List;
+
+public class OrderedList extends UnorderedList<OrderedListItem> {
+
+    private int elementCount;
+
+    public OrderedList() {
+        elementCount = 0;
+    }
+
+    public OrderedList(List<OrderedListItem> items) {
```

Comment:
Use generic type in this and the other methods.

```suggestion
    public OrderedList(List<T> items) {
```